tests/behaviour/contracts/external_function_inputs/static_array_return_index.sol

```diff
@@ -4,10 +4,29 @@ pragma solidity ^0.8.0;
 
 contract WARP {
 
+struct StrucDef {
+    uint8 x;
+    uint256 y;
+}
 
-    function testReturnIndex(int16[3] memory listA) pure external returns (int16) {
-        //int16[2] memory listB = [listA[1], listA[2]];
+    function testA(int16[3] memory listA) pure external returns (int16) {
+         return listA[2];
+    }
+
+    function testB(int16[3] memory listA) pure public returns (int16) {
         return listA[2];
     }
+    // Would return y but warp at this stage does not return whole structs.
```

Comment:
Would it make sense to change the function to `returns (uint8, uint256)` and the return statement to `return (listA[2].x, listA[2].y)`?

Reply:
uint256 is a struct. At this point we cannot return a whole struct, but I could change it to `return (listA[2].x, listA[2].y.low, listA[2].y.high)`

Reply:
Resolved